spec/support/helpers/features_helper.rb

```diff
@@ -47,11 +47,10 @@ def enter_visitor_information(options = {})
     end
   end
 
-  def select_slots(how_many = 3)
```

Comment:
I think it would be good to preserve the ability to select multiple slots, and perhaps also to test that all three are properly selected!